CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Changelog
 
 ## master
 
+- Add NSButton Introspection
```

Comment:
Would be nice to put `.introspectButton` here, since it's in line with other changelog entries.

Reply:
Done